industry_benchmarks/utils/results_cleanup.py

```diff
@@ -0,0 +1,372 @@
+                    results_dir
+                )
+
+                # Now we should check if the dir exists
+                if not results_dir.is_dir():
+                    print("Can't find results directory, skipping")
+                    continue
+
+            structural_analysis_data = results["unit_results"][proto_key]["outputs"][
+                "structural_analysis"
+            ]
+
+            # save structural analysis data
+            print("Saving structural analysis data")
+            np.savez_compressed(
+                results_dir / "structural_analysis_data.npz",
+                protein_RMSD=np.asarray(
+                    structural_analysis_data["protein_RMSD"], dtype=np.float32
+                ),
+                ligand_RMSD=np.asarray(
+                    structural_analysis_data["ligand_RMSD"], dtype=np.float32
+                ),
+                ligand_wander=np.asarray(
+                    structural_analysis_data["ligand_wander"], dtype=np.float32
+                ),
+                protein_2D_RMSD=np.asarray(
+                    structural_analysis_data["protein_2D_RMSD"], dtype=np.float32
+                ),
+                time_ps=np.asarray(
+                    structural_analysis_data["time(ps)"], dtype=np.int32
+                ),
+            )
+
+            # remove structural_analysis data stuffed into unit results
+            del results["unit_results"][proto_key]["outputs"]["structural_analysis"]
+
+            # Now we sub sample the traj and save reporter data
+            simulation = results_dir / "simulation.nc"
+            checkpoint = Path("checkpoint.chk")
+            hybrid_pdb = results_dir / "hybrid_system.pdb"
+            # TODO better name?
+            outfile = results_dir / "energy_replica_state.npz"
+            out_traj = results_dir / "out"
+            print("Subsampling trajectory and saving energy data")
+            extract_data(simulation, checkpoint, hybrid_pdb, outfile, out_traj)
+
+            # remove structural_analysis data stuffed into protocol_result
+            # and remove ligand + pdb
+            # this data is duped in structural_analysis_data.npz
+            print("Shrinking result JSON")
+            del results["protocol_result"]["data"][result_key][0]["outputs"][
+                "structural_analysis"
+            ]
+            del results["protocol_result"]["data"][result_key][0]["inputs"]["stateA"]
+            del results["protocol_result"]["data"][result_key][0]["inputs"]["stateB"]
+            del results["protocol_result"]["data"][result_key][0]["inputs"][
+                "ligandmapping"
+            ]
+
+            print("Saving JSON")
+            with open(json_file, "w") as f:
+                json.dump(results, f)
+
+            # Now we delete files we don't need anymore
+            print("Deleting trajectory files")
+            os.remove(simulation)
+            os.remove(checkpoint)
```

Comment:
```suggestion
            os.remove(results_dir / checkpoint)
```

Reply:
Ah yes, this was working okay when it was the full path, but for the reporter it just needs the file name I so I think when I fixed the doc string I changed this to match (not thinking that os.remove needs the full path)

Reply:
can you test it now?

Reply:
have done, works thanks!